addons/wowl/doc/dropdown_component.md

```diff
@@ -0,0 +1,186 @@
+# Dropdown Component
+
+## Overview
+
+As dropdowns are common in Odoo, we decided to make a generic dropdown component.
+
+It contains all the logic of what you can usually expect a dropdown to behave.
+
+We made the choice to develop this simple component with as few style as possible, therefore it is your responsibility to make yours look awesome 😎!
+
+## Behind the scene
+
+A `<Dropdown/>` component is simply a `<div class="o_dropdown"/>` having a `<button/>` next to an unordered list (`<ul/>`). The button controls if the list is attached to the DOM or not.
+
+A `<DropdownItem/>` is simply a list item (`<li/>`) transporting a payload (which you'll receive back in a custom `dropdown-item-selected` event when the item get clicked).
```

Comment:
A `<DropdownItem/>` is simply a list item (`<li/>`). On click, you can ask this item to return you a payload (which you'll receive back in a custom `dropdown-item-selected` event). This payload is an object, so feel free to put anything you want in it. Most likely, you will return an id to know which item was clicked.

Reply:
Most likely, you will use ids as payloads to know which item was clicked.